src/Lecture1.hs

```diff
@@ -50,7 +51,11 @@ Explanation: @sumOfSquares 3 4@ should be equal to @9 + 16@ and this
 is 25.
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-sumOfSquares x y = error "TODO!"
+sumOfSquares :: Integer -> Integer -> Integer
+sumOfSquares x y = square x  + square  y
+    where
+        square :: Integer -> Integer
+        square n = n ^ 2
```

Comment:
Great idea on the helper function! 👏🏻 

I see that GHC outputs some warnings here. I think you can resolve them by putting type annotation to a number.
```suggestion
        square n = n ^ (2 :: Int)
```
Alternatively, you can implement this function differently without relying on type annotations.

```suggestion
        square n = n * n
```